Update command line parameter documents (#10806)
pingcap/docs-cn: command-line-flags-for-tidb-configuration.md

>
> 需谨慎使用 `*` 符号，因为它可能引入安全风险，允许来自任何 IP 的客户端自行汇报其 IP 地址。另外，它可能会导致部分直接连接 TiDB 的内部组件无法使用，例如 TiDB Dashboard。

> > **注意：**

Reviewer: Shall we combine the preceding warning and this note to make the text more readable?
![image](https://user-images.githubusercontent.com/89768198/190415148-5a45a501-9bdc-47a5-a291-b61bf1d1c694.png)

Author: 同上

Reviewer: 同上：
![image](https://user-images.githubusercontent.com/89768198/190564343-3a3f1fb4-d494-47bf-a5ef-002b3236edb0.png)

>
> 如果使用 AWS 的 Network Load Balancer (NLB) 并开启 PROXY 协议，需要设置 NLB 的 `target group` 属性：将 `proxy_protocol_v2.client_to_server.header_place` 设为 `on_first_ack`。同时向 AWS 的 Support 提工单开通此功能的支持。由于 AWS NLB 在开启 PROXY 协议后，默认只在客户端发送数据之后才会发送 PROXY 的报文，同时，在客户端发送数据包之前，服务器端发送的任何数据包都会在内网被丢弃，所以，对于 MySQL 协议来说，客户端无法获取服务器端的握手报文，因此会一直阻塞到客户端超时。

## `--proxy-protocol-header-timeout`

+ PROXY 协议请求头读取超时时间
+ 默认：5
+ 单位：秒

> > **警告：**

Reviewer: Shall we combine the note and the warning:
![image](https://user-images.githubusercontent.com/89768198/190412753-983b65f6-49a2-4820-8dd3-375059cb79c9.png)

Author: 我看还是分开吧，毕竟是2个意思。

Reviewer: 可能我没有说清楚，不是说把 note 和 warning 合为一句话，而是合在一个 note 下面作为两个并列的注意项，目前上面一个警告，下面一个注意，阅读体验不是很好。英文的我已经先这样改了，你看看：
![image](https://user-images.githubusercontent.com/89768198/190563927-672dde02-7dba-4718-933b-935df93e08a3.png)

>

> > - 自 v6.3.0 起，该参数被废弃。因为自 v6.3.0 起，读取 PROXY 协议报头的操作会在第一次读取网络数据时进行，废弃该参数可避免影响首次读取网络数据时设置的超时时间。

Reviewer: ```suggestion
> 自 v6.3.0 起，该参数被废弃。因为自 v6.3.0 起，读取 PROXY 协议报头的操作会在第一次读取网络数据时进行，废弃该参数可避免影响首次读取网络数据时设置的超时时间。
```

>
> 需谨慎使用 `*` 符号，因为它可能引入安全风险，允许来自任何 IP 的客户端自行汇报其 IP 地址。另外，它可能会导致部分直接连接 TiDB 的内部组件无法使用，例如 TiDB Dashboard。

> **注意：**
>

> > 如果使用 AWS 的 NLB 并开启 PROXY Protocol 协议，需要设置 NLB 的 `target group` 属性：`proxy_protocol_v2.client_to_server.header_place` 到 `on_first_ack`，并同时向AWS 的 Support 提工单开通此功能的支持。由于 AWS NLB 在开启 PROXY Protocol 协议的默认行为为只有在客户端发送数据之后才会发送 PROXY Protocol 的报文，同时在客户端发送数据包之前，服务器端发送的任何数据包都会在内网被丢弃，所以对于 MySQL 协议来说，客户端无法获取服务器端的握手报文，因此会一直阻塞到客户端超时。

Reviewer 2: NBL 是负载均衡器吗？能否写全称呢？

Author: Updated

> > 如果使用 AWS 的 Network Load Balancer (简称 NLB) 并开启 PROXY Protocol 协议，需要设置 NLB 的 `target group` 属性：`proxy_protocol_v2.client_to_server.header_place` 到 `on_first_ack`，并同时向AWS 的 Support 提工单开通此功能的支持。由于 AWS NLB 在开启 PROXY Protocol 协议的默认行为为只有在客户端发送数据之后才会发送 PROXY Protocol 的报文，同时在客户端发送数据包之前，服务器端发送的任何数据包都会在内网被丢弃，所以对于 MySQL 协议来说，客户端无法获取服务器端的握手报文，因此会一直阻塞到客户端超时。

Reviewer 2: ```suggestion
> 如果使用 AWS 的 Network Load Balancer (简称 NLB) 并开启 PROXY 协议，需要设置 NLB 的 `target group` 属性：将 `proxy_protocol_v2.client_to_server.header_place` 设为 `on_first_ack`。同时向 AWS 的 Support 提工单开通此功能的支持。由于 AWS NLB 在开启 PROXY 协议后，默认只在客户端发送数据之后才会发送 PROXY 的报文，同时在客户端发送数据包之前，服务器端发送的任何数据包都会在内网被丢弃，所以对于 MySQL 协议来说，客户端无法获取服务器端的握手报文，因此会一直阻塞到客户端超时。
```

## `--proxy-protocol-header-timeout`

> 

Reviewer 2: ```suggestion

> **注意：**
>
> - 自 v6.3.0 起，该参数被废弃。因为自 v6.3.0 起，读取 PROXY Protocol 报头的操作会在第一次读取网络数据时进行，废弃该参数可避免影响首次读取网络数据时设置的超时时间。

```

>
> 不要将该参数配置为 `0`。除非特殊情况，一般使用默认值即可。

> **弃用：**
>

> > 新版本中，读取 PROXY Protocol 报头的动作会在第一次读取网络数据时进行，因此为避免影响首次读取网络数据时设置的超时时间，此参数被弃用。

Reviewer 2: 这里的「新版本」是哪个版本呢？

Author: 我看应该是在 6.3 版本开始引入了对应的PR，这里已经改为 6.3.0 版本之后

> > **弃用：**
> >
> > 在 TiDB 6.3.0 版以后，读取 PROXY Protocol 报头的动作会在第一次读取网络数据时进行，因此为避免影响首次读取网络数据时设置的超时时间，此参数被弃用。
> 

Reviewer 2: ```suggestion
```
